src/attestation.rs

```diff
@@ -23,6 +23,7 @@ pub(crate) enum AttestationFormat {
     AndroidKey,
     AndroidSafetyNet,
     FIDOU2F,
+    Apple,
```

Comment:
This line should be AppleAnonymous

Reply:
Got it—this is despite the corresponding fmt string being "Apple"?

Reply:
That needs to change too then. The spec calls it AppleAnonymous so it should always be AppleAnonymous in all contexts.

Reply:
According to the [docs](https://webkit.org/blog/11312/meet-face-id-and-touch-id-for-the-web/)

```

$$attStmtType //= (
                       fmt: "apple",
                       attStmt: appleStmtFormat
                   )
```

it seems like the string should be "apple."

Reply:
Ahh that's lovely and confusing as all the other references to it are Apple Anonymous. In that case, you are correct :)

Reply:
Indeed, not the most straightforward setup I've seen. To clarify, the variant name should still change to `AppleAnonymous`, correct?